src/pybind/mgr/mgr_module.py

```diff
@@ -1172,7 +1172,18 @@ def get_context(self) -> object:
     def notify(self, notify_type: str, notify_id: str) -> None:
         """
         Called by the ceph-mgr service to notify the Python plugin
-        that new state is available.
+        that new state is available.  This method is *only* called for
+        notify_types that are listed in the NOTIFY_TYPES string list
+        member of the module class.
+
+        The following events are defined:
+           mon_map - mon_map change
+           pg_summary - new MMgrDigest from mon, possible pg summary change
+           health - new MMgrDigest from mon, possible health state change
+           clog - cluster log event
+           osd_map - osdmap change
+           fs_map - fs map change
+           command - CLI command completed
```

Comment:
let's also document those types that were disabled. Otherwise no one is gonna know.

```suggestion
           command - CLI command completed
           service_map - servicemap change. (currently deactivated. See Mgr.cc) 
           mon_status - mon status change. (currently deactivated. See Mgr.cc) 
           perf_schema_update (currently deactivated. See DaemonServer.cc) 
```

Alternatively, we could add an enum and properly document them in the docs:

```python
# mgr_module.py
from enum import Enum
class NotifyTypes(str, Enum):
    mon_map = 'mon_map'  #: mon_map change
    pg_summary = 'pg_summary'  #: new MMgrDigest from mon, possible pg summary change
    health = 'health'  #: new MMgrDigest from mon, possible health state change
    clog = 'clog'  #: cluster log event
    osd_map = 'osd_map'  #: osdmap change
    fs_map = 'fs_map'  #: fs map change
    command = 'command'  #: CLI command completed
    service_map = 'service_map'  #: servicemap change. (currently deactivated. See Mgr.cc) 
    mon_status = 'mon_status'  #: mon status change. (currently deactivated. See Mgr.cc) 
    perf_schema_update = 'perf_schema_update'  #: (currently deactivated. See DaemonServer.cc) 
```
plus
```diff
diff --git a/doc/mgr/modules.rst b/doc/mgr/modules.rst
index 9fb3b87ae3f..52c746e08be 100644
--- a/doc/mgr/modules.rst
+++ b/doc/mgr/modules.rst
@@ -380,6 +380,8 @@ simply return from this function without doing anything.
 
 .. automethod:: MgrModule.notify
 
+.. autoclass:: NotifyTypes
+
 Accessing RADOS or CephFS
 -------------------------
```

Reply:
Yes!  much better this way